.github/workflows/_linux_ut.yml

```diff
@@ -141,3 +148,81 @@ jobs:
             test_cmd="${test_cmd} test_xpu.py"
           fi
           eval $test_cmd
+
+      - name: Upload Inductor XPU UT Data
+        uses: actions/upload-artifact@v4
+        with:
+          name: Inductor-XPU-UT-Data-${{ github.event.pull_request.number || github.sha }}
+          path: ${{ github.workspace }}/ut_log
+
+      - name: UT Test Results Check
+        if: ${{ inputs.expected_fail_num }}
+        shell: bash
+        run: |
```

Comment:
Can we abstract this step as a script and put it into `.github/scripts`? and in this step, we not only check the test pass or not, but also show the failure cases and logs directly